yarn.lock

```diff
@@ -17154,7 +17154,7 @@ storybook-chromatic@^2.2.2:
     tree-kill "^1.1.0"
     uuid "^3.3.2"
 
-storybook-react-router@^1.0.8:
+storybook-react-router@1.0.8:
```

Comment:
??

Reply:
Just did a `sudo rm -rf node_modules/ && yarn` to update deps.